freqtrade/optimize/backtesting.py

```diff
@@ -218,21 +218,18 @@ def _get_ohlcv_as_lists(self, processed: Dict[str, DataFrame]) -> Dict[str, Tupl
         """
         # Every change to this headers list must evaluate further usages of the resulting tuple
         # and eventually change the constants for indexes at the top
-        headers = ['date', 'buy', 'open', 'close', 'sell', 'low', 'high']
+        headers = ['date', 'buy', 'open', 'close', 'sell', 'low', 'high', 'buy_tag']
```

Comment:
This will fail for all strategies that don't define a buy signal, because the column will not exist.

That part was handled in L230...

Reply:
hm - seems i missed that line - and was slightly distracted by the failing tests :laughing: